master/internal/rm/multirm/multirm.go

```diff
@@ -547,3 +547,8 @@ func (m *MultiRMRouter) fanOutRMCommand(f func(rm.ResourceManager) error) error
 	}
 	return nil
 }
+
+// SmallerValueIsHigherPriority returns true if smaller priority values indicate a higher priority level.
+func (m *MultiRMRouter) SmallerValueIsHigherPriority() (bool, error) {
+	return false, nil
```

Comment:
Just to confirm, are we setting this to `false` because multiRM implies the cluster must be all k8s RMs, and k8s RMs use higher value -> higher priority?

Reply:
Yeaa I like this a lot! _Super_ clean way to handle this Kristine! 
Can we add a comment mentioning this?

Reply:
Yes, great idea!

Reply:
I was writing a comment, and felt bad about stating that kind of assumption so I rewrote it to call the underlying RM.   😅

Reply:
Ahh lol comments keep us honest 😆. FWIW, I do think you initial method of returning `false` was clean as well, but it's definitely more intuitive / easy to understand from an outside perspective to call the underlying RM, nice work!